.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
```

Comment:
All those `.idea` changes should be removed from the pull request.

Note that git supports a local ignore list which will apply to all your repositories and is your best bet to avoid such things ever getting included in a branch.